.envrc

```diff
@@ -85,12 +85,12 @@ It's recommended to use pyenv - please refer to https://docs.sentry.io/developme
 }
 
 advice_install_sentry() {
-    info "To install sentry, please type: make install-py-dev"
+    info "To install sentry, please type: source .venv/bin/activate && make install-py-dev"
```

Comment:
how about we add a function like

```sh
advice_activate_venv() {
	if [ -z ${PYTHON_VENV} ]; then
		info "Activate your virtualenv with: source .venv/bin/activate"
		return 1
	fi
	return 0
}

advice_install_sentry() {
    advice_activate_venv() && info "To install sentry, please type: make install-py-dev"
}
```

My bash probably isn't completely right here, but you get the idea